R/staging.txt

```diff
@@ -0,0 +1,13 @@
+# My understanding is that all I need to do is paste the link to the book's webpage. 
+# Hardcover out in October, eBook in November, happy for this to wait until then if that's preferable.
+# I will also add `description`, `github-repo`, and `cover-image` information. 
+# Please let me know what else might be needed since this looks atypical. Thanks!
+https://www.springer.com/us/book/9783030211257

```

Comment:
All I need is a link here. Any other lines shouldn't be included. The link must point to the web version of an open and free book written and published with **bookdown**. The website bookdown.org is not a place to exhibit arbitrary books, but only books written with **bookdown** and can be read online for free.

Reply:
Oh, I wrote this with **bookdown** but it _cannot_ be read online for free. Thanks for the clarification.

Reply:
Perhaps you could try to convince your editor at Springer :) Chapman & Hall and O'Reilly have adopted this model of "free online + hardcopy for purchase", as you can see from the list of books on bookdown.org. We haven't had a Springer book published in this way yet, and would love to see the first example!

Reply:
Excellent, it's only fitting that the first example at Springer comes from yet another R luminary. Keep up the fantastic work, all of you!